homeassistant/components/tuya/vacuum.py

```diff
@@ -178,7 +185,9 @@ def pause(self, **kwargs: Any) -> None:
 
     def return_to_base(self, **kwargs: Any) -> None:
         """Return device to dock."""
-        self._send_command([{"code": DPCode.MODE, "value": "chargego"}])
+        if self.find_dpcode(DPCode.SWITCH_CHARGE, prefer_function=True):
+            self._send_command([{"code": DPCode.SWITCH_CHARGE, "value": True}])
+        self._send_command([{"code": DPCode.MODE, "value": TUYA_MODE_RETURN_HOME}])
```

Comment:
We can just send both commands in the same call. Unknown commands are ignored anyways.